### PR TITLE
docs: Add Mono Repo Usage

### PR DESCRIPTION
**Pull request by makdeniz**
Signed-off-by: Murat Akdeniz makdeniz@msn.com

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [X] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [x] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: docs/operator-manual/notifications/triggers.md
Comment (crenshaw-dev) [inline docs/operator-manual/notifications/triggers.md:70]: Just suggesting a couple grammar tweaks:

```
When one repo is used to sync multiple applications, the `oncePer: app.status.sync.revision` field will trigger a notification for each commit. For mono repos, the better approach will be using `oncePer: app.status.operationState.syncResult.revision` statement. This way a notification will be sent only for a particular Application's revision.
```

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]
Thanks for the PR! Just requesting a tweak.

Can I get a second review from someone more familiar w/ notifications?

**Comment by BeyondEvil** [comment]
This fixed the issue we were having with multiple applications in the same manifest repo.

When application A had changes, notifications were triggered for application B. Changing the `oncePer` as per this suggestion fixed it. 🙏

File: docs/operator-manual/notifications/triggers.md
Comment (brunocascio) [inline docs/operator-manual/notifications/triggers.md:70]: this save my life

**Comment by brunocascio** [comment]
This is helpful with applicationset-based repos where you attach annotations dynamically on every application and only the changed app should send the notification

**Review by crenshaw-dev — APPROVED** [review]

**Comment by crenshaw-dev** [comment]
I'm not a notifications expert, but makes sense to me. Thanks @makdeniz!

**Pull request merged**
